### PR TITLE
fix(css): make "show password" button background white on blur

### LINKED ISSUE #6196: Safari/Chrome: background stays grey after pressing the "eye"
child of https://github.com/mozilla/fxa-content-server/issues/6261

Found by @vladikoff in https://github.com/mozilla/fxa-content-server/pull/6195#issuecomment-389209645

> Unrelated I guess I noticed in Safari and Chrome, after we press show the button stays "focused" grey:
> aaa

This is because the "show-password" label becomes focused. Perhaps after releasing it, we should blur the element?

### PR DESCRIPTION
**Pull request by brizental**
Fixes #6196 

I explained what I did here on https://github.com/mozilla/fxa-content-server/issues/6196#issuecomment-394891321

r? @vladikoff

### CODE REVIEW

File: app/styles/modules/_password-row.scss [hunk 1]
```diff
@@ -201,19 +201,13 @@
 
 .show-password-label:hover,
 .show-password-label:active,
-.show-password-label:focus,
-.show-password:hover + .show-password-label,
-.show-password:active + .show-password-label,
-.show-password:focus + .show-password-label {
+.show-password-label:focus {
```
Comment (vladikoff) [inline app/styles/modules/_password-row.scss]: 👍

**Comment by vladikoff** [comment]
@brizental Awesome work, thank you!

File: app/styles/modules/_password-row.scss (hunk 1 above)
Comment (shane-tomlinson) [reply app/styles/modules/_password-row.scss]: Will this work with keyboard navigation?

Comment (vladikoff) [reply app/styles/modules/_password-row.scss]: @shane-tomlinson we don't support keyboard on those things AFAIK, users just jump between password fields

**Pull request merged**